guide/14-deep-learning/edge-detection.ipynb

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
<p>Replace title with the title of the guide. We need this for better search engine indexing and for browsers to title the tabs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I have added a title to the guide and renamed the notebook with the same name. Is that what you meant?